### PR TITLE
Create all build files when releasing app

### PR DESCRIPTION
**Pull request by snopoke**
This is to avoid having to create them on demand which can cause issues if you have a lot of users all updating at the same time.

### CODE REVIEW

**Comment by orangejenny** [comment]
I'm a little surprised this is done on release instead of on build. I'd expect people would usually want to download and test out their app before they mark it as released.

File: corehq/apps/app_manager/views/releases.py [hunk 1]
```diff
@@ -170,6 +171,8 @@ def release_build(request, domain, app_id, saved_app_id):
     app_post_release.send(Application, application=saved_app)
 
     if is_released:
+        if saved_app.build_profiles and domain_has_privilege(domain, privileges.BUILD_PROFILES):
```
Comment (emord) [inline corehq/apps/app_manager/views/releases.py]: Is it possible to test if all the builds have already been created? although I suppose not many have build profiles

**Review by emord — APPROVED** [review]
@orangejenny I think since this is specifically for apps with multiple profiles it's reasonable that someone would test one profile but not all of them (4 for icds)

**Comment by snopoke** [comment]
The task does check if they've been created by looking for the
`profile.xml` file for each build profile

**Pull request merged**